src/Storage/Controllers/DataController.cs

```diff
@@ -130,6 +130,7 @@ public async Task<ActionResult<DataElement>> Delete(int instanceOwnerPartyId, Gu
                 return await InitiateDelayedDelete(instance, dataElement);
             }
 
+            dataElement.LastChangedBy = User.GetUserOrOrgId();
```

Comment:
should we also set it in the case of delayed delete?

Reply:
Done